cyences_app_for_splunk/default/macros.conf

```diff
@@ -846,6 +846,10 @@ definition = search *
 definition = index IN (wincerts)
 iseval = 0
 
+[cs_windows_disable/uninstall_critical_agents_filter]
```

Comment:
Are we validating this is Critical softwares by any condition?? If not let's remove Critical keyword